server/region_handler.go

```diff
@@ -768,6 +874,125 @@ func (t *regionHandlerTool) getMvccByStartTs(startTS uint64, startKey, endKey []
 	}
 }
 
+func (t *regionHandlerTool) getMvccByIdxValue(idx table.Index, values url.Values, idxCols []*model.ColumnInfo, handleStr string) (*kvrpcpb.MvccGetByKeyResponse, error) {
+	idxRow, err := t.formValue2DatumRow(values, idxCols)
+	if err != nil {
+		return nil, err
```

Comment:
Do we need to add the error trace? Some places have not added in this PR.

Reply:
Add error traces for the whole file.